docs/User Survey

```diff
@@ -0,0 +1,13 @@
+<style> #disqus_thread, #__comments { display: none } </style>
+
+<h2> Take the brainlife.io user survey </h2>
+
+We are inviting all registered brainlife.io users take a survey about their experience with the platform. The survey is anonymous, unless users choose to share their contact information.
+
+We are asking users to participate in this survey because understanding our users’ needs and experiences with brainlife.io will help our team improve the platform to meet the needs of all students and researchers who want to use it.
+
+This survey will run until July 1 and take approximately 5 to 10 minutes to complete. Before you begin, you will read a Study Information Sheet about this study. 
```

Comment:
This survey will take approximately 5 to 10 minutes to complete and run until July 1.